nixos/modules/services/video/photonvision.nix

```
@@ -0,0 +1,64 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.photonvision;
+in
+{
+  options = {
+    services.photonvision = {
+      enable = lib.mkEnableOption (lib.mdDoc "Enable PhotonVision");
+
+      package = lib.mkPackageOption pkgs "PhotonVision" { default = [ "photonvision" ]; };
```

```suggestion
      package = lib.mkPackageOption pkgs "photonvision" { };
```

This should be enough